main.rb

```diff
@@ -0,0 +1,155 @@
+require_relative 'person'
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'book'
+require_relative 'rental'
+class Main
+  def initialize()
+    @person = []
+    @books = []
+    @rental = []
+  end
```

Comment:
- [ ] Great job so far. However, I would like to draw your attention to a few adjustments that I think would help improve the structure of your files and code as mentioned in the project requirements. In the project requirements, it was stated specifically that you should create a file called `app.rb` which will be where you're going to implement the below requirements: 
<img width="874" alt="Screenshot 2023-10-21 at 01 03 16" src="https://github.com/MayPyone/School-Library/assets/107192076/a1e74b2a-b737-439c-b071-d0916c177c2e">


- [ ] Also, your `main.rb` isn't supposed to be a class but rather a method that will be invoked at the end of your file. The `main.rb` should be the entry point for your app and should contain code related to the following requirements:
<img width="1069" alt="Screenshot 2023-10-21 at 01 09 04" src="https://github.com/MayPyone/School-Library/assets/107192076/6baa1f32-37fe-4bfd-8607-b058260b78dc">

Currently, You have all the necessary methods, and everything is working fine as expected but kindly make sure to adhere to project requirements by making changes as I have mentioned above.

Happy coding! 😊🚀👨‍💻

Reply:
well resolved! ✔️